main.js

```diff
@@ -1,3 +1,36 @@
-import './style.css';
+function addOrRemoveTooltip(){
+    return this.title !== '' ? addTooltip(this) : removeTooltip(this)
+}
+function addTooltip(elem){
+    const title = elem.title;
+    elem.title = '';
+    elem.setAttribute("tooltip", title);
+    const tooltipWrap = document.createElement("div");
+    tooltipWrap.className = 'tooltip';
+    tooltipWrap.appendChild(document.createTextNode(title));
+    const padding = 35;
+    const linkProps = elem.getBoundingClientRect();
+    const tooltipProps = tooltipWrap.getBoundingClientRect();
+    const topPos = linkProps.top - (tooltipProps.height + padding);
+    tooltipWrap.setAttribute('style','top:'+topPos+'px;'+'left:'+linkProps.left+'px;')
+    elem.parentNode.insertBefore(tooltipWrap, elem);
+}
+function removeTooltip(elem){
+    elem.title = elem.getAttribute("tooltip");
+    elem.removeAttribute("tooltip");
+    document.querySelector(".tooltip").remove();
```

Comment:
You will remove first random tooltip like so. What if you have 10 buttons with tooltips on click?

Reply:
I replaced it by `queryAll`